### PR TITLE
Feature textures

### LINKED ISSUE #133: QR Code - Show
Show the QR code on the second screen.

This is a subtask of #130 QR Code.

### PR DESCRIPTION
**Pull request by WytzeElhorst**
Adds textures to the game and opens window with qr code

this closes #133 but the main purpose of this pr is to have textures for our demo

### CODE REVIEW

**Comment by WytzeElhorst** [comment]
turn out textures did not load in correctly yet

File: src/main/java/nl/tudelft/contextproject/controller/GameController.java [hunk 1]
```diff
@@ -168,18 +209,18 @@ void updateEntities(float tpf) {
...
-					break;
-				case NEW:
-					addDrawable(e);
-					e.setState(EntityState.ALIVE);
-					e.update(tpf);
-					break;
-				default:
-					e.update(tpf);
-					break;
+			case DEAD:
```
Comment (Geweldig) [inline src/main/java/nl/tudelft/contextproject/controller/GameController.java]: incorrect spacing

File: src/main/java/nl/tudelft/contextproject/model/level/MazeTile.java [hunk 2]
```diff
@@ -42,25 +44,41 @@ public MazeTile(int x, int y, TileType type) {
...
-				this.height = 3;
-				this.color = ColorRGBA.Blue;
-				break;
-			case CORRIDOR:
-				this.height = 0;
-				this.color = ColorRGBA.Red;
-				break;
-			default:
-				throw new IllegalArgumentException("Invalid TileType: " + type);
+		case FLOOR:
```
Comment (Geweldig) [inline src/main/java/nl/tudelft/contextproject/model/level/MazeTile.java]: incorrect format

File: src/main/java/nl/tudelft/contextproject/model/level/MazeTile.java [hunk 3]
```diff
@@ -42,25 +44,40 @@ public MazeTile(int x, int y, TileType type) {
...
+				if (!(Main.getInstance().getAssetManager() == null)) {
+					this.texture = Main.getInstance().getAssetManager().loadTexture("Textures/grasstexture.png");
+				} else {
+					this.texture = null;
+				}
 				break;
 			default:
 				throw new IllegalArgumentException("Invalid TileType: " + type);
-		}
+			}
```
Comment (Geweldig) [inline src/main/java/nl/tudelft/contextproject/model/level/MazeTile.java]: should be moved one tab to the left

**Comment by Geweldig** [comment]
After my issues have been addressed I think this can be merged.

File: src/main/java/nl/tudelft/contextproject/model/level/MazeTile.java [hunk 4]
```diff
@@ -42,25 +44,40 @@ public MazeTile(int x, int y, TileType type) {
...
+					this.texture = Main.getInstance().getAssetManager().loadTexture("Textures/grasstexture.png");
+				} else {
+					this.texture = null;
+				}
 				break;
 			case WALL:
 				this.height = 3;
-				this.color = ColorRGBA.Blue;
+				this.color = ColorRGBA.Gray;
+				if (!(Main.getInstance().getAssetManager() == null)) {
```
Comment (ChielBruin) [inline src/main/java/nl/tudelft/contextproject/model/level/MazeTile.java:61]: This should not be needed

File: src/main/java/nl/tudelft/contextproject/model/level/MazeTile.java [hunk 5]
```diff
@@ -83,7 +100,8 @@ public Spatial getSpatial() {
 		mat.setColor("Specular", ColorRGBA.White);
 		mat.setFloat("Shininess", 64f);
 		mat.setColor("Ambient", color);
-		this.spatial.setMaterial(mat);
+		mat.setTexture("LightMap", texture);
+		this.spatial.setMaterial(mat);                   // set the cube's material
```
Comment (ChielBruin) [inline src/main/java/nl/tudelft/contextproject/model/level/MazeTile.java]: This comment could be removed

File: src/main/java/nl/tudelft/contextproject/model/level/MazeTile.java (hunk 4 above)
Comment (WytzeElhorst) [inline src/main/java/nl/tudelft/contextproject/model/level/MazeTile.java:61]: it should not, but ClientServletTest fails without it

File: src/main/java/nl/tudelft/contextproject/Main.java [hunk 6]
```diff
@@ -57,7 +61,15 @@ public static void main(String[] args) {
...
-		
+		if (java.awt.Desktop.isDesktopSupported()) {
+			   try {
+			    java.awt.Desktop.getDesktop().browse(new URI("file://" + new File("qrcode.png").getAbsolutePath().replace("\\", "/")));
+			   } catch (IOException ex) {
+			    ex.printStackTrace();
+			   } catch (URISyntaxException ex) {
+			    ex.printStackTrace();
+			   }
+			  }
```
Comment (Taeir) [inline src/main/java/nl/tudelft/contextproject/Main.java]: Should be removed, is relocated to the proper location in another branch

**Comment by Geweldig** [comment]
If @Taeir agrees this can be merged.

**Comment by Taeir** [comment]
Looks good now, merging it.

**Pull request merged**